libpromises/expand.c

```diff
@@ -632,7 +632,7 @@ void BundleResolvePromiseType(EvalContext *ctx, const Bundle *bundle, const char
             for (size_t i = 0; i < SeqLength(pt->promises); i++)
             {
                 Promise *pp = SeqAt(pt->promises, i);
-#if 1
+#if 0
```

Comment:
Thanks Nick. BTW you are the last person I was expecting to find this gem and comment here. :-)